vignettes/introduction.Rmd

```diff
@@ -25,6 +26,173 @@ knitr::opts_chunk$set(
 library(mmrm)
 ```
 
+# Common usage
+
+[A minimal call of mmrm()](https://openpharma.github.io/mmrm/main/reference/mmrm.html),
+consisting of only formula and data arguments will produce an object of class 
+`mmrm`, `mmrm_fit`, and `mmrm_tmb`. Here we fit a mmrm
+model with `us` (unstructured) covariance structure specified, as well as the defaults
+of `reml = TRUE` and `optimizer = 'automatic'`.
+
+```{r common-usage-modelcall}
+fit <- mmrm(
+  formula = FEV1 ~ RACE + ARMCD * AVISIT + us(AVISIT | USUBJID),
+  data = fev_data
+)
+```
+
+Printing the object will show you output which should be familiar to anyone who 
+has used any popular modeling functions such as `stats::lm()`, `stats::glm()`,
+`glmmTMB::glmmTMB()`, and `lme4::nlmer()`. From this print out we see the function call,
+the data used, the covariance structure with number of variance parameters, as well 
+as the likelihood method, and model deviance achieved. Additionally the user is 
+provided a printout of the estimated coefficients and the model convergence information.
+
+```{r common-usage-print}
+print(fit)
+```
+
+# Common changes
+
+From the high-level `mmrm()` interface, common changes to the default function 
+call can be specified.
+
+## REML or ML
+
+Users can specify if REML should be used (default) or if ML should be used in 
+optimization.
+
+```{r common-changes-reml}
+fit_ml <- mmrm(
+  formula = FEV1 ~ RACE + ARMCD * AVISIT + us(AVISIT | USUBJID),
+  data = fev_data,
+  reml = FALSE
+)
+
+print(fit_ml)
+```
+
+## Optimizer
+
+Users can specify which optimizer should be used, changing from the default of 
+"automatic" selection, which starts with `"L-BFGS-B"` and proceeds through the 
+other choices if optimization fails to converge. Other choices are "BFGS", "CG", and
+"nlminb". 
```

Comment:
consider just writing this as code, e.g. `automatic` instead of "automatic". I usually do this in the function documentation e.g. for strings.

Reply:
thanks, will do